helm/chart-operator-chart/Chart.yaml

```diff
@@ -1,4 +1,4 @@
 apiVersion: v1
 description: A Helm chart for the chart-operator
 name: chart-operator-chart
-version: 0.2.0
+version: 0.2.1
```

Comment:
Why does this change now?

Reply:
This was something Ross mentioned in his review further up: `Can you also make sure you bump the Helm chart version to 0.2.1.`
I am not 100% why it has to be that exact version but I believe it needs to be updated to control the installation of `chart-operator` into guest clusters by `cluster-operator`.

Reply:
We need to bump version for every change, to make sure they get pushed to a new channel. Ross bumped to `0.2.0` in his previous PR.

Reply:
Yep, if the version is not changed cluster-operator won't rollout the changes. More important from my POV, you can accidentally overwrite the version on the stable channel, so that if for some reason it gets redeployed, a version from a PR can be accidentally used.

Reply:
That sounds like I will just forget about it when I have to do a PR here. What can we do about this manual step?

Reply:
Yep this is know :/ right now we keep eagle eyes on it. But IMO we shouldn't wait too long to automate this.